health-sdk/README.md

```diff
@@ -3,54 +3,47 @@
 Gini Health SDK for Android
 ===========================
 
-// TODO: update documentation to align with the style of capture sdk (https://github.com/gini/gini-capture-sdk-android/blob/main/README.md)
-// and bank sdk (https://github.com/gini/gini-pay-bank-sdk-android/blob/main/README.md)
+The Gini Health SDK for Android provides all the UI and functionality needed to use the Gini Health API in your app to
+extract payment and health information from invoices. The payment information can be reviewed and then the invoice can
+be paid using any available payment provider app (e.g., banking app).
 
-Installation
-------------
-
-To install add our Maven repo to the root build.gradle file and add it as a dependency to your app
-module's build.gradle.
-
-build.gradle:
+The Gini Health API provides an information extraction service for analyzing health invoices. Specifically, it extracts
+information such as the document sender or the payment relevant information (amount to pay, IBAN, etc.). In addition it
+also provides a secure channel for sharing payment related information between clients. 
 
-```
-repositories {
-    maven {
-        url 'https://repo.gini.net/nexus/content/repositories/open
-    }
-}
-```
+Documentation
+-------------
 
-app/build.gradle:
+TODO: update urls before releasing the documentation
 
-```
-dependencies {
-    implementation 'net.gini:gini-pay-business-sdk:1.0.5'
-}
-```
+* [Integration Guide](http://developer.gini.net/gini-health-sdk-android/html/)
+* [Reference Docs](http://developer.gini.net/gini-health-sdk-android/ginicapture/dokka/index.html)
 
 Example Apps
----
+------------
 
-### Health
+### Health Example App
 
-The health example app is in the `:health-sdk:example-app` module.
-It needs `health-sdk/example-app/local.properties` with credentials:
+You can see a sample usage of the Gini Health SDK in the `:health-sdk:example-app` module. 
+
+It requires Gini Health API credentials which are injected automatically if you create this file `health-sdk/example-app/local.properties` with the following properties:
 ```
 clientId=*******
 clientSecret=*******
 ```
 
-### Bank
+### Bank Example App
 
 In order to pass the Requirements a bank app needs to be installed on the device.
 
+TODO: use Gini Bank SDK urls after it's released
+
 An example bank app is available in the Gini Pay Bank SDK's
 [repository](https://github.com/gini/gini-pay-bank-sdk-android) called
 [`appcreenapi`](https://github.com/gini/gini-pay-bank-sdk-android/tree/main/appscreenapi).
 
-## License
+License
+-------
 
-Gini Health SDK is available under a commercial license.
+The Gini Health SDK for Android is available under a commercial license.
```

Comment:
Maybe it's better to use the same statement about the licence like you've done:
The Gini Health API Library for Android is licensed under an MIT/Expat license.

Reply:
🙈 I have to change it too.